go-controller/pkg/ovn/policy.go

```diff
@@ -195,7 +195,7 @@ func (oc *Controller) updateStaleDefaultDenyACLNames(npType knet.PolicyType, gre
 	p := func(item *nbdb.ACL) bool {
 		return item.ExternalIDs[defaultDenyPolicyTypeACLExtIdKey] == string(npType) && // default-deny-policy-type:Egress or default-deny-policy-type:Ingress
 			strings.Contains(item.Match, gressSuffix) && // Match:inport ==	@ablah80448_egressDefaultDeny or Match:inport == @ablah80448_ingressDefaultDeny
-			!strings.Contains(*item.Name, arpAllowPolicySuffix) && // != name: namespace_ARPallowPolicy
+			!strings.Contains(item.Match, "arp") && // != match: (arp || nd)
```

Comment:
Why not define a constant for the full match that is used both there and where the ACL is created. That way if it is changed in one place in the future, we won't miss the change here and will be reminded to check if this logic is still valid.

Reply:
great point Jaime! let me use constants..